packages/playwright/src/reporters/list.ts

```diff
@@ -31,13 +31,11 @@ class ListReporter extends BaseReporter {
   private _resultIndex = new Map<TestResult, string>();
   private _stepIndex = new Map<TestStep, string>();
   private _needNewLine = false;
-  private readonly _liveTerminal: string | boolean | undefined;
   private _printSteps: boolean;
 
   constructor(options: { omitFailures?: boolean, printSteps?: boolean } = {}) {
     super(options);
-    this._printSteps = options.printSteps || !!process.env.PW_TEST_DEBUG_REPORTERS_PRINT_STEPS;
-    this._liveTerminal = process.stdout.isTTY || !!process.env.PWTEST_TTY_WIDTH;
+    this._printSteps = isTTY && (options.printSteps || !!process.env.PW_TEST_DEBUG_REPORTERS_PRINT_STEPS);
```

Comment:
I think stops printing steps outside of tty, which is a semantic change.

Reply:
yeah, but is there a way to?